docs/modules/colbert/pages/index.adoc

```diff
@@ -0,0 +1,90 @@
+= Introduction to ColBERT
+
+ColBERT stands for "Contextualized Late Interaction over BERT".
+
+"Contextualized Late Interaction" describes a unique method of interacting with Stanford University's https://arxiv.org/abs/2004.12832[BERT]{external-link-icon} model.
+
+ColBERT is a machine learning retrieval model that improves the computational efficiency and contextual depth of information retrieval tasks.
+
+*TL;DR:*
+
+1. BERT embeds text chunks as matrices of token-level vectors, enabling much deeper context matching than a single vector embedding per chunk.
+2. BERT manages this additional depth by pre-processing documents and queries into uniform lengths with the https://huggingface.co/learn/nlp-course/en/chapter6/6[Wordpiece] tokenizer, ideal for batch processing on GPUs.
+3. "Contextualized Late Interaction" first retrieves the top-k chunks with the highest similarity scores to the query token.
+The top-k chunks are then sorted again. The query token is compared to every token in the chunk to rank the chunks by the highest aggregate similarity score.
```

Comment:
change to query `tokens` (plural).  change `is` to `are`, etc... to make sense for plural query tokens.